charts/scaffold/Chart.lock

```diff
@@ -0,0 +1,15 @@
+dependencies:
```

Comment:
same question @sabre1041 , is that ok

Reply:
Probably good to mention is that it is easy to manage this see https://github.com/sigstore/helm-charts/pull/300 on how folks can update when bumping the dependency versions.